code/__HELPERS/game.dm

```diff
@@ -512,6 +512,8 @@
 		return
 	winset(C, "mainwindow", "flash=5")
 
+/proc/holiday_active(holiday)
+	return SSevent.holidays && SSevent.holidays[holiday]

```

Comment:
Should probably give off a warning if holidays aren't set up yet instead of failing silently